mod/wet4/languages/en.php

```diff
@@ -147,7 +147,7 @@
 
     'group:accept:invitation' => 'Accept<span class="wb-inv">group invite to "%s"</span>',
     'group:decline:invitation' => 'Decline<span class="wb-inv">group invite to "%s"</span>',
-
+    'groups:toomanyfriends' => 'Looks like you\'re too popular! Your colleagues list can\'t be loaded because you have over 1,000 colleagues.',
```

Comment:
I feel it would be valuable to let the user know they can still use the "Find member" option to invite users. @LemieuxGen